src/Applications/conf.yaml

```diff
@@ -26,7 +26,11 @@
 # generate_multimedia [-g]        bool      generate multimedia
 # order_strategy                  str       arbitrary, element_id, fewest_migrations, small_objects, largest_objects
 
-criterion: 0
```

Comment:
You should have changed the description/documentation in YAML file as well, so it corresponds to the different data format in conf file

criterion [-c <tc>]   int(0-3)  transfer criterion: 
- 0: lower total work, 
- 1: tempered load only (default),
- 2: strict localizer, 
- 3: relaxed localizer

Reply:
No. It is taking all existing criteria, but the description must be done as follows:
```
criterion:
  name: "LowerTotalWork"
  parameters:
    alpha: 0.
    beta: 0.001
```
They are to be called by name, instead of by index.